controller/test/sensor_tests/sensors_test.cpp

```diff
@@ -72,42 +91,115 @@ TEST(SensorTests, FullScaleReading) {
   // versus what the original pressure waveform was
   for (auto p : pressures) {
     SCOPED_TRACE("Pressure " + std::to_string(p.kPa()));
-    Hal.test_setAnalogPin(AnalogPin::PATIENT_PRESSURE,
-                          MPXV5004_PressureToVoltage(p));
-    Hal.test_setAnalogPin(AnalogPin::INFLOW_PRESSURE_DIFF,
-                          MPXV5004_PressureToVoltage(kPa(3.0)));
-    Hal.test_setAnalogPin(AnalogPin::OUTFLOW_PRESSURE_DIFF,
-                          MPXV5004_PressureToVoltage(kPa(1.0)));
-
-    auto readings = sensors.GetSensorReadings();
+
+    auto readings = update_readings(seconds(0.0f), p, p, p, sensors);
     float pressurePatient = cmH2O(readings.patient_pressure_cm_h2o).kPa();
-    EXPECT_NEAR(pressurePatient, p.kPa(), COMPARISON_TOLERANCE);
+    EXPECT_NEAR(pressurePatient, p.kPa(), COMPARISON_TOLERANCE_PRESSURE);
 
     // Inhalation and exhalation should match because they are fed with the same
-    // pressure waveform
-    auto expected_flow = Sensors::PressureDeltaToFlow(kPa(3.0)) -
-                         Sensors::PressureDeltaToFlow(kPa(1.0));
-    EXPECT_NEAR(expected_flow.cubic_m_per_sec(),
-                ml_per_min(readings.flow_ml_per_min).cubic_m_per_sec(),
-                COMPARISON_TOLERANCE_FLOW);
+    // pressure waveform ==> total flow should be 0
```

Comment:
What's the reason for returning to the previous, more "vacuous" assertion here? If we're verifying correct measurement of inflow and outflow in the other test, then might as well remove it here completely?

Reply:
yes that makes sense.